cmd/root/root.go

```diff
@@ -51,10 +49,7 @@ func New(ctx context.Context) *cobra.Command {
 			return err
 		}
 
-		logger := log.GetLogger(ctx)
-		logger.Info("start",
-			slog.String("version", build.GetInfo().Version),
-			slog.String("args", strings.Join(os.Args, ", ")))
+		log.Infof(ctx, "Starting execution with version: %s, arguments: %s", build.GetInfo().Version, strings.Join(os.Args, ", "))
```

Comment:
This is not equivalent: with `slog.String` we can pull out the version programmatically in JSON mode.